account_invoice_import_invoice2data/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     'author': 'Akretion,Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/edi',
     'depends': ['account_invoice_import'],
-    'external_dependencies': {'python': ['invoice2data']},
+    'external_dependencies': {'python': ['invoice2data==0.3.5']},
```

Comment:
Runbot is not liking this, can you set it to invoice2data and keep the change only on requirements?

Reply:
the problem seems that runbot and travis are working differently :dizzy_face:

Reply:
> the problem seems that runbot and travis are working differently

Yes, that is exactly the problem.
If we have ` 'external_dependencies': {'python': ['invoice2data']},` runbot works, but travis install the newest invoice2data, and then it fails on python 3.6 f-strings....

The only fix is probaly to get runbot and travis (MQT) to have the same understanding of external_dependencies and requirements